MyPostgresDriver can't instantiate in play-scala example

Trying to run the play-scala example, get Configuration Error "Cannot connect to database [default]" with stacktrace ending in the following:

Caused by: java.lang.ClassNotFoundException: util.MyPostgresDriver$

I have postgres running with a "test" database created as configured in the example's application.conf. Did I miss some setup?

--- a/examples/play-slick-example/build.sbt
+++ b/examples/play-slick-example/build.sbt
@@ -10,7 +10,7 @@ description := "slick-pg play integration example project"
 
 libraryDependencies ++= Seq(
   specs2,
-  "com.typesafe.play" %% "play-slick" % "1.0.0",
+  "com.typesafe.play" %% "play-slick" % "1.0.1",
   "com.typesafe.play" %% "play-slick-evolutions" % "1.0.0",
   "com.github.tminglei" %% "slick-pg" % "0.9.0",
   "org.postgresql" % "postgresql" % "9.4-1201-jdbc41",

